### ISSUE TITLE
pytest package missing

### ISSUE THREAD

**Issue by benjaminbolling** [bwheelz36/ParticlePhaseSpace#148]
pytest package missing from [setup configuration](https://github.com/bwheelz36/ParticlePhaseSpace/blob/main/setup.cfg).

**Comment by bwheelz36** [comment]
Hi @benjaminbolling - this is intentional - the setup.cfg file contains the dependencies used for packaging, but not for development. The development dependencies are listed in the `dev_requirements.txt` file. 

For my newer packaged, I started using `pyproject.toml` which allows one to seperate build/dev depencies - but I never got this working with setup.cfg, which is why I stuck to the requirements file. If you know of a good way to do it with setup.cfg files I'd be happy to hear it! 

Anyway, all this was never actually stated anywhere, so I updated the landing page of the documentation in https://github.com/bwheelz36/ParticlePhaseSpace/commit/2b673958682e64bc8b61a691eca1ad5de54ecca6

Does this resolve this concern for you?
